site/content/docs/development/multi-cluster.md

```diff
@@ -0,0 +1,35 @@
+# Multi-Cluster
+
+## Overview
+It is often desirable to separate development and production environments; this can be achieved with multiple clusters.
+Cartographer helps pave a path to production across clusters with [ClusterSupplyChains](reference/workload.md#clustersupplychain)
+and [ClusterDeliveries](reference/deliverable.md#clusterdelivery), leveraging [GitOps](https://www.gitops.tech/).
+
+A ClusterSupplyChain can be installed in the development cluster to manage the path from source code to Kubernetes
+configuration in a Git repository. Meanwhile, a ClusterDelivery can be installed in the production cluster, which
+will pick up that configuration from the Git repository to be deployed and tested.
+
+![Multi-Cluster](../img/multi-cluster.jpg)
+
+### Example PR based flow
+0. Developer commits new source code
+1. ClusterSupplyChain in the development cluster generates Kubernetes configuration and commits it to a
+   git repository/branch (i.e. staging/feature-a)
+2. "Staging environment maintainer" opens a PR and merges staging/feature-a into staging
+3. ClusterDelivery in the staging cluster picks up the merged PR and deploys the Kubernetes configuration to be tested and commits
+   it to a git repository/branch (i.e. production/feature-a)
```

Comment:
I've always preferred to have a separate git repository for environments / clusters and "promote"  into these.  The main reason for that is folks tend to have different owners or approvers for different clusters.  Keeping them separate means we can control RBAC via the repo for who is allowed to merge a PR to staging, and a different set of users that own production merges.  It also helps avoid accidentally merging to the wrong branch which could mean production getting changes that were meant for staging.